AUTHORS

```diff
@@ -32,12 +32,11 @@ New contributors to libchewing:
   Shaform <shaform@gmail.com>
   Chocobo1
   Perry Thompson <contact@ryper.org>
-  Yen Chi Hsuan <yan12125@gmail.com>
   Peter Dave Hello <hsu@peterdavehello.org>
   ShengYenPeng <peter03192003@gmail.com>
   cwlin <halboof@gmail.com>
   Billy Su <billy4195.su@gmail.com>
-  Chih-Hsuan Yen <yan12125@gmail.com>
+  Chih-Hsuan Yen (Yen Chi Hsuan) https://github.com/yan12125
```

Comment:
Can you leave only one identity? If you don't tend to keep the email, simply drop it.